diff --git a/src/data/home-data/MenuData.ts b/src/data/home-data/MenuData.ts
index 8edca97..69b3b40 100644
--- a/src/data/home-data/MenuData.ts
+++ b/src/data/home-data/MenuData.ts
@@ -38,9 +38,6 @@ const menu_data: MenuItem[] = [
         sub_menus: [
             
             { link: "/properties", title: "All Properties" },
-            { link: "/properties", title: "Browse by Type" },
-            { link: "/properties", title: "Property Details" },
-            { link: "/properties", title: "Featured Opportunities" },
             
         ],
     },
