diff --git a/src/app/agency/page.tsx b/src/app/agency/page.tsx
deleted file mode 100644
index b07acf0..0000000
--- a/src/app/agency/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import Agency from "@/components/inner-pages/agency/agency";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Agency",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <Agency />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/blog_03/page.tsx b/src/app/blog_03/page.tsx
deleted file mode 100644
index e9ae9bf..0000000
--- a/src/app/blog_03/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import BlogThree from "@/components/blogs/blog-three";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Blog Three",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <BlogThree />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/compare/page.tsx b/src/app/compare/page.tsx
deleted file mode 100644
index 59fcb90..0000000
--- a/src/app/compare/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import Compare from "@/components/inner-pages/compare";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Compare",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <Compare />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/home-two/page.tsx b/src/app/home-two/page.tsx
deleted file mode 100644
index 61feb3f..0000000
--- a/src/app/home-two/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import HomeTwo from "@/components/homes/home-two";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-  title: "Property Planet — Home Two",
-};
-const index = () => {
-  return (
-    <Wrapper>
-      <HomeTwo />
-    </Wrapper>
-  )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/pricing_01/page.tsx b/src/app/pricing_01/page.tsx
deleted file mode 100644
index 555b23a..0000000
--- a/src/app/pricing_01/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import PricingOne from "@/components/inner-pages/pricing/pricing-one";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Pricing One",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <PricingOne />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/project_01/page.tsx b/src/app/project_01/page.tsx
deleted file mode 100644
index 8e26360..0000000
--- a/src/app/project_01/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ProjectOne from "@/components/inner-pages/projects/project-one";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-  title: "Property Planet — Project One",
-};
-const index = () => {
-  return (
-    <Wrapper>
-      <ProjectOne />
-    </Wrapper>
-  )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/project_02/page.tsx b/src/app/project_02/page.tsx
deleted file mode 100644
index d6ee4eb..0000000
--- a/src/app/project_02/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ProjectTwo from "@/components/inner-pages/projects/project-two";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-  title: "Property Planet — Project Two",
-};
-const index = () => {
-  return (
-    <Wrapper>
-      <ProjectTwo />
-    </Wrapper>
-  )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/project_03/page.tsx b/src/app/project_03/page.tsx
deleted file mode 100644
index 01f45f7..0000000
--- a/src/app/project_03/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ProjectThree from "@/components/inner-pages/projects/project-three";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Project Three",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <ProjectThree />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/project_04/page.tsx b/src/app/project_04/page.tsx
deleted file mode 100644
index 0909bde..0000000
--- a/src/app/project_04/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ProjectFour from "@/components/inner-pages/projects/project-four";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Project Four",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <ProjectFour />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/service_01/page.tsx b/src/app/service_01/page.tsx
deleted file mode 100644
index 97d77b2..0000000
--- a/src/app/service_01/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ServiceOne from "@/components/inner-pages/services/service-one";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Service One",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <ServiceOne />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/app/service_02/page.tsx b/src/app/service_02/page.tsx
deleted file mode 100644
index 2314bdb..0000000
--- a/src/app/service_02/page.tsx
+++ /dev/null
@@ -1,15 +0,0 @@
-import ServiceTwo from "@/components/inner-pages/services/service-two";
-import Wrapper from "@/layouts/Wrapper";
-
-export const metadata = {
-   title: "Property Planet — Service Two",
-};
-const index = () => {
-   return (
-      <Wrapper>
-         <ServiceTwo />
-      </Wrapper>
-   )
-}
-
-export default index
\ No newline at end of file
diff --git a/src/components/blogs/blog-three/BlogThreeArea.tsx b/src/components/blogs/blog-three/BlogThreeArea.tsx
deleted file mode 100644
index 8fefa42..0000000
--- a/src/components/blogs/blog-three/BlogThreeArea.tsx
+++ /dev/null
@@ -1,91 +0,0 @@
-"use client"
-import inner_blog_data from '@/data/inner-data/BlogData'
-import Link from "next/link"
-import { useState } from "react";
-import ReactPaginate from 'react-paginate';
-
-const categories: string[] = ["all", "apartments", "villa", "mortgage", "sale", "home", "flat"];
-
-const BlogThreeArea = () => {
-   const blog = inner_blog_data.filter((items) => items.page === "blog_3");
-
-   const [selectedCategory, setSelectedCategory] = useState("all"); // Step 1
-
-   const itemsPerPage = 6;
-   const [itemOffset, setItemOffset] = useState(0);
-   const endOffset = itemOffset + itemsPerPage;
-   const filteredBlog = selectedCategory === "all" ? blog : blog.filter((item) => item.category === selectedCategory); // Step 3
-   const currentItems = filteredBlog.slice(itemOffset, endOffset);
-   const pageCount = Math.ceil(filteredBlog.length / itemsPerPage);
-
-   const handlePageClick = (event: any) => {
-      const newOffset = (event.selected * itemsPerPage) % filteredBlog.length;
-      setItemOffset(newOffset);
-   };
-
-   const handleCategoryClick = (category: string) => {
-      setSelectedCategory(category); // Step 2
-      setItemOffset(0); // Reset offset when category changes
-   };
-
-
-   return (
-      <div className="blog-section-three mt-130 xl-mt-100 mb-150 xl-mb-100">
-         <div className="container">
-            <div className="blog-filter-nav">
-               <ul className="style-none d-flex justify-content-center flex-wrap isotop-menu-wrapper">
-                  {categories.map((category) => (
-                     <li
-                        key={category}
-                        className={selectedCategory === category ? "is-checked" : ""}
-                        onClick={() => handleCategoryClick(category)}
-                     >
-                        {category.charAt(0).toUpperCase() + category.slice(1)} {/* Capitalize first letter */}
-                     </li>
-                  ))}
-               </ul>
-            </div>
-
-            <div className=" row isotop-gallery-2-wrapper pt-60 lg-pt-40">
-               {currentItems.map((item) => (
-                  <div key={item.id} className="col-lg-6">
-                     <div className="isotop-item villa sale">
-                        <article className="blog-meta-one mb-70 lg-mb-40">
-                           <figure className={`post-img border-25 position-relative m0 ${item.class_name}`}>
-                              <Link href="/blog_details" className="stretched-link date tran3s">{item.date}</Link>
-                           </figure>
-                           <div className="post-data">
-                              <div className="post-info"><Link href="/blog_details">{item.info_name}</Link> {item.info_time} min</div>
-                              <div className="d-flex justify-content-between align-items-sm-center flex-wrap">
-                                 <Link href="/blog_details" className="blog-title">
-                                    <h4>{item.title}</h4>
-                                 </Link>
-                                 <Link href="/blog_details" className="read-btn rounded-circle d-flex align-items-center justify-content-center tran3s">
-                                    <i className="bi bi-arrow-up-right"></i>
-                                 </Link>
-                              </div>
-                           </div>
-                        </article>
-                     </div>
-                  </div>
-               ))}
-            </div>
-
-            <div className="pt-20 text-center">
-               <ReactPaginate
-                  breakLabel="..."
-                  nextLabel={<i className="fa-regular fa-chevron-right"></i>}
-                  onPageChange={handlePageClick}
-                  pageRangeDisplayed={5}
-                  pageCount={pageCount}
-                  previousLabel={<i className="fa-regular fa-chevron-left"></i>}
-                  renderOnZeroPageCount={null}
-                  className="pagination-two d-inline-flex align-items-center justify-content-center style-none"
-               />
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default BlogThreeArea
diff --git a/src/components/blogs/blog-three/index.tsx b/src/components/blogs/blog-three/index.tsx
deleted file mode 100644
index f8623f5..0000000
--- a/src/components/blogs/blog-three/index.tsx
+++ /dev/null
@@ -1,19 +0,0 @@
-import BreadcrumbOne from "@/components/common/breadcrumb/BreadcrumbOne"
-import FooterFour from "@/layouts/footers/FooterFour"
-import HeaderOne from "@/layouts/headers/HeaderOne"
-import BlogThreeArea from "./BlogThreeArea"
-import FancyBanner from "@/components/common/FancyBanner"
-
-const BlogThree = () => {
-   return (
-      <>
-         <HeaderOne style={true} />
-         <BreadcrumbOne title="Blog Full Width" link="#" link_title="Pages" sub_title="Blog" style={true} />
-         <BlogThreeArea />
-         <FancyBanner />
-         <FooterFour />
-      </>
-   )
-}
-
-export default BlogThree
diff --git a/src/components/inner-pages/agency/agency/AgencyArea.tsx b/src/components/inner-pages/agency/agency/AgencyArea.tsx
deleted file mode 100644
index 7255736..0000000
--- a/src/components/inner-pages/agency/agency/AgencyArea.tsx
+++ /dev/null
@@ -1,89 +0,0 @@
-"use client"
-import agency_data from "@/data/inner-data/AgecyData";
-import NiceSelect from "@/ui/NiceSelect"
-import Image from "next/image"
-import Link from "next/link"
-import { useState } from "react";
-import { Rating } from "react-simple-star-rating";
-import ReactPaginate from 'react-paginate';
-
-import paginateIcon from "@/assets/images/icon/icon_46.svg";
-
-const AgencyArea = () => {
-
-   const selectHandler = (e: any) => { };
-   const agency = agency_data;
-
-   const itemsPerPage = 9;
-   const [itemOffset, setItemOffset] = useState(0);
-   const endOffset = itemOffset + itemsPerPage;
-   const currentItems = agency.slice(itemOffset, endOffset);
-   const pageCount = Math.ceil(agency.length / itemsPerPage);
-   // click to request another page.
-   const handlePageClick = (event: any) => {
-      const newOffset = (event.selected * itemsPerPage) % agency.length;
-      setItemOffset(newOffset);
-   };
-
-   return (
-      <div className="agency-section mt-130 xl-mt-100 mb-150 xl-mb-100">
-         <div className="container">
-            <div className="listing-header-filter d-sm-flex justify-content-between align-items-center mb-35">
-               <div>Showing <span className="color-dark fw-500">{itemOffset + 1}–{itemOffset + currentItems.length}</span> of <span
-                  className="color-dark fw-500">{agency_data.length}</span> results</div>
-               <div className="d-flex align-items-center xs-mt-20">
-                  <div className="short-filter d-flex align-items-center">
-                     <div className="fs-16 me-2">Short by:</div>
-                     <NiceSelect
-                        className="nice-select rounded-0"
-                        options={[
-                           { value: "01", text: "Pupular" },
-                           { value: "02", text: "Best Rating" },
-                           { value: "03", text: "Price Low" },
-                           { value: "04", text: "Price High" },
-                        ]}
-                        defaultCurrent={0}
-                        onChange={selectHandler}
-                        name=""
-                        placeholder="" />
-                  </div>
-               </div>
-            </div>
-            <div className="row gx-xxl-5">
-               {currentItems.map((item) => (
-                  <div key={item.id} className="col-lg-4 col-sm-6">
-                     <div className="card-style-nine position-relative z-1 mb-50 lg-mb-30 wow fadeInUp" data-wow-delay="0.1s">
-                        <div className="tag bg-white position-absolute text-uppercase">{item.tag}</div>
-                        <div className="logo-wrapper d-flex align-items-center justify-content-center">
-                           <Image src={item.thumb} alt="" className="lazy-img" />
-                        </div>
-                        <div className="text-center pt-35">
-                           <h6 className="agency-name">{item.title}</h6>
-                           <ul className="rating style-none d-flex justify-content-center align-items-center">
-                              <li><Rating initialValue={item.rating} allowFraction={true} size={16} readonly={true} /></li>
-                              <li><span>{item.rating}</span></li>
-                           </ul>
-                           <p>{item.desc}</p>
-                           <Link href="/agency_details" className="btn-eight w-100 rounded-0 tran3s">View Listing</Link>
-                        </div>
-                     </div>
-                  </div>
-               ))}
-            </div>
-
-            <ReactPaginate
-               breakLabel="..."
-               nextLabel={<Image src={paginateIcon} alt="" className="ms-2" />}
-               onPageChange={handlePageClick}
-               pageRangeDisplayed={3}
-               pageCount={pageCount}
-               previousLabel={<Image src={paginateIcon} alt="" className="ms-2" />}
-               renderOnZeroPageCount={null}
-               className="pagination-one square d-flex align-items-center justify-content-center style-none pt-40"
-            />
-         </div>
-      </div>
-   )
-}
-
-export default AgencyArea
diff --git a/src/components/inner-pages/agency/agency/index.tsx b/src/components/inner-pages/agency/agency/index.tsx
deleted file mode 100644
index 0707a14..0000000
--- a/src/components/inner-pages/agency/agency/index.tsx
+++ /dev/null
@@ -1,19 +0,0 @@
-import BreadcrumbThree from "@/components/common/breadcrumb/BreadcrumbThree"
-import HeaderFour from "@/layouts/headers/HeaderFour"
-import AgencyArea from "./AgencyArea"
-import FooterFour from "@/layouts/footers/FooterFour"
-import FancyBanner from "@/components/common/FancyBanner"
-
-const Agency = () => {
-   return (
-      <>
-         <HeaderFour />
-         <BreadcrumbThree title="Agency" link="#" link_title="Pages" sub_title="Agency" style={true} />
-         <AgencyArea />
-         <FancyBanner/>
-         <FooterFour/>
-      </>
-   )
-}
-
-export default Agency
diff --git a/src/components/inner-pages/compare/CompareArea.tsx b/src/components/inner-pages/compare/CompareArea.tsx
deleted file mode 100644
index 985f422..0000000
--- a/src/components/inner-pages/compare/CompareArea.tsx
+++ /dev/null
@@ -1,150 +0,0 @@
-import Image, { StaticImageData } from "next/image";
-import Link from "next/link"
-
-import compareLogo from "@/assets/images/logo/logo_08.svg";
-
-import compareThumb_1 from "@/assets/images/listing/img_66.jpg";
-import compareThumb_2 from "@/assets/images/listing/img_67.jpg";
-import compareThumb_3 from "@/assets/images/listing/img_68.jpg";
-
-interface Datatype {
-   id: number;
-   tag: string;
-   thumb: StaticImageData;
-   title: string;
-   desc: string;
-}
-
-const compare_data: Datatype[] = [
-   {
-      id: 1,
-      tag: "For Sell",
-      thumb: compareThumb_1,
-      title: "Blueberry villa",
-      desc: "2972 Westheimer Rd. Santa Ana, Illinois 85486",
-   },
-   {
-      id: 2,
-      tag: "For Sell",
-      thumb: compareThumb_2,
-      title: "Orkit Flat",
-      desc: "2118 Thornridge Cir. Syracuse, Connecticut 35624",
-   },
-   {
-      id: 3,
-      tag: "For Sell",
-      thumb: compareThumb_3,
-      title: "Modu Apartment",
-      desc: "1901 Thornridge Cir. Shiloh, Hawaii 81063",
-   },
-]
-
-interface TableData {
-   table_head: string;
-   table_data: string[];
-}[];
-
-const table_row_data: TableData[] = [
-   {
-      table_head: "Bedrooms",
-      table_data: ["3", "2", "3"],
-   },
-   {
-      table_head: "Bath",
-      table_data: ["2", "1", "1"],
-   },
-   {
-      table_head: "Price",
-      table_data: ["$1,75,320", "$1,42,720", "$1,82,000"],
-   },
-   {
-      table_head: "Property Type",
-      table_data: ["Villa", "Flat", "Apartment"],
-   },
-   {
-      table_head: "Property ID",
-      table_data: ["h2180", "h3218", "h4322"],
-   },
-   {
-      table_head: "Parking",
-      table_data: ["1", "2", "0"],
-   },
-   {
-      table_head: "Lot Size",
-      table_data: ["2,710 sq.ft", "1,810 sq.ft", "3,470 sq.ft"],
-   },
-   {
-      table_head: "Features",
-      table_data: ["Ac/Heating, Laundry, Garden, Elevator, Pet Friendly, Wifi, Refrigerator", "Ac/Heating, Garden, Wifi, Pet Friendly, Refrigerator", "Ac/Heating, Laundry, Garden, Elevator, Pet Friendly, Wifi, Refrigerator, Fireplace, Swimming Pool"],
-   },
-   {
-      table_head: "Zoning",
-      table_data: ["Residential", "Residential", "Residential"],
-   },
-]
-
-const CompareArea = () => {
-   return (
-      <div className="compare-section mt-150 xl-mt-100 mb-150 xl-mb-100">
-         <div className="container">
-            <div className="row align-items-center">
-               <div className="col-md-6">
-                  <div className="title-one">
-                     <h3>Compare Listing</h3>
-                  </div>
-               </div>
-               <div className="col-xl-3 col-lg-4 col-md-5 ms-auto">
-                  <ul className="style-none d-flex align-items-center action-btns sm-mt-40">
-                     <li className="me-auto fw-500 color-dark"><i className="fa-sharp fa-regular fa-share-nodes me-2"></i> Share</li>
-                     <li><Link href="#" className="d-flex align-items-center justify-content-center tran3s"><i className="fa-light fa-heart"></i></Link></li>
-                     <li><Link href="#" className="d-flex align-items-center justify-content-center tran3s"><i className="fa-light fa-bookmark"></i></Link></li>
-                     <li><Link href="#" className="d-flex align-items-center justify-content-center tran3s"><i className="fa-light fa-circle-plus"></i></Link></li>
-                  </ul>
-               </div>
-            </div>
-
-            <div className="compare-table mt-60 lg-mt-40">
-               <div className="table-responsive table-bg bg-white">
-                  <table className="table">
-                     <thead>
-                        <tr>
-                           <td><Image src={compareLogo} alt="" className="lazy-img mt-50" /></td>
-                           {compare_data.map((item) => (
-                              <td key={item.id}>
-                                 <div className="listing-card-one style-two shadow-none">
-                                    <div className="img-gallery">
-                                       <div className="position-relative overflow-hidden">
-                                          <div className="tag fw-500">{item.tag}</div>
-                                          <Link href="#" className="fav-btn tran3s"><i className="fa-light fa-heart"></i></Link>
-                                          <Link href="#" className="d-block">
-                                             <Image src={item.thumb} className="w-100" alt="..." />
-                                          </Link>
-                                       </div>
-                                    </div>
-                                    <div className="property-info pt-10">
-                                       <Link href="#" className="title tran3s">{item.title}</Link>
-                                       <div className="address">{item.desc}</div>
-                                    </div>
-                                 </div>
-                              </td>
-                           ))}
-                        </tr>
-                     </thead>
-
-                     <tbody>
-                        {table_row_data.map((item, index) => (
-                           <tr key={index}>
-                              <th>{item.table_head}</th>
-                              {item.table_data.map((data, i) => <td key={i}>{data}</td>)}
-                           </tr>
-                        ))}
-                     </tbody>
-                  </table>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default CompareArea
diff --git a/src/components/inner-pages/compare/index.tsx b/src/components/inner-pages/compare/index.tsx
deleted file mode 100644
index 363185c..0000000
--- a/src/components/inner-pages/compare/index.tsx
+++ /dev/null
@@ -1,19 +0,0 @@
-import BreadcrumbThree from "@/components/common/breadcrumb/BreadcrumbThree"
-import HeaderFour from "@/layouts/headers/HeaderFour"
-import FooterFour from "@/layouts/footers/FooterFour"
-import CompareArea from "./CompareArea"
-import FancyBanner from "@/components/common/FancyBanner"
-
-const Compare = () => {
-   return (
-      <>
-         <HeaderFour />
-         <BreadcrumbThree title="Compare" link="#" link_title="Pages" sub_title="Compare" style={false} />
-         <CompareArea />
-         <FancyBanner />
-         <FooterFour />
-      </>
-   )
-}
-
-export default Compare
diff --git a/src/components/inner-pages/pricing/pricing-one/PricingOneArea.tsx b/src/components/inner-pages/pricing/pricing-one/PricingOneArea.tsx
deleted file mode 100644
index 0eea7a5..0000000
--- a/src/components/inner-pages/pricing/pricing-one/PricingOneArea.tsx
+++ /dev/null
@@ -1,82 +0,0 @@
-"use client"
-import pricing_data from "@/data/inner-data/PricingData";
-import Link from "next/link";
-import { useState } from "react";
-
-const tab_title: string[] = ["Monthly", "Yearly",];
-const list: string[] = ["All Operating Supported", "Multiple User", "Refund", "12 months duration", "Live Chat", "Send invite via Link"];
-
-const PricingOneArea = () => {
-
-   const selectHandler = (e: any) => { };
-   const [activeTab, setActiveTab] = useState(0);
-
-   const handleTabClick = (index: any) => {
-      setActiveTab(index);
-   };
-
-   return (
-      <div className="pricing-section-one mt-150 xl-mt-100">
-         <div className="container">
-            <div className="row">
-               <div className="col-lg-10 m-auto">
-                  <div className="title-one text-center mb-40 lg-mb-30 wow fadeInUp">
-                     <h3>Unbeatable prices, no contracts, simple & easy</h3>
-                     <p className="fs-24">Try Team plan features for 14 days · No credit card required</p>
-                  </div>
-               </div>
-            </div>
-
-            <div className="pr-table-one">
-               <nav className="pricing-nav-one d-flex justify-content-center">
-                  <div className="nav nav-tabs" role="tablist">
-                     {tab_title.map((tab, index) => (
-                        <button key={index} onClick={() => handleTabClick(index)} className={`nav-link ${activeTab === index ? "active" : ""}`} id="buy-tab" type="button">{tab}</button>
-                     ))}
-                  </div>
-               </nav>
-               <div className="discount-text mt-15 text-center">Save 30% on Annual plan</div>
-
-               <div className="dot-bg-wrapper mt-60 lg-mt-40">
-                  <div className="tab-content">
-                     {pricing_data.filter((items) => items.page === "pricing_1").map((pricing, index) => (
-                        <div key={index} className={`tab-pane show ${activeTab === index ? "active" : ""}`} id="monthly">
-                           <div className="main-bg d-flex flex-wrap justify-content-end position-relative">
-                              <div className="left-panel d-none d-lg-block">
-                                 <ul className="style-none">
-                                    {list.map((list, i) => <li key={i}>{list}</li>)}
-                                 </ul>
-                              </div>
-                              {pricing.pricing_data.map((item) => (
-                                 <div key={item.id} className={`pr-column-wrapper ${item.class_name}`}>
-                                    <div className="pr-header text-center">
-                                       <div className="plan text-uppercase">{item.plan}</div>
-                                       <strong className="price fw-500">{item.price}</strong>
-                                       <p className="fs-16">{item.desc}</p>
-                                    </div>
-                                    <ul className="style-none text-center">
-                                       {item.icon_details.map((icon, i) => (
-                                          <li key={i}>
-                                             <span className="fw-500 color-dark">All Operating Supported</span>
-                                             <div className={`icon d-flex align-items-center justify-content-center rounded-circle ${icon.icon_class}`}>
-                                                <i className={icon.icon}></i></div>
-                                          </li>
-                                       ))}
-                                    </ul>
-                                    <div className="pr-footer text-center">
-                                       <Link href="#" className="btn-twelve sm">{item.btn}</Link>
-                                    </div>
-                                 </div>
-                              ))}
-                           </div>
-                        </div>
-                     ))}
-                  </div>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default PricingOneArea
diff --git a/src/components/inner-pages/pricing/pricing-one/index.tsx b/src/components/inner-pages/pricing/pricing-one/index.tsx
deleted file mode 100644
index 3c21b83..0000000
--- a/src/components/inner-pages/pricing/pricing-one/index.tsx
+++ /dev/null
@@ -1,21 +0,0 @@
-import BreadcrumbOne from "@/components/common/breadcrumb/BreadcrumbOne"
-import FooterFour from "@/layouts/footers/FooterFour"
-import HeaderOne from "@/layouts/headers/HeaderOne"
-import PricingOneArea from "./PricingOneArea"
-import FancyBannerCommon from "@/components/common/FancyBanner"
-import FancyBanner from "../../projects/project-one/FancyBanner"
-
-const PricingOne = () => {
-   return (
-      <>
-         <HeaderOne style={true} />
-         <BreadcrumbOne title="Pricing Plan" link="#" link_title="Pages" sub_title="Pricing" style={true} />
-         <PricingOneArea />
-         <FancyBanner/>
-         <FancyBannerCommon style={false} />
-         <FooterFour />
-      </>
-   )
-}
-
-export default PricingOne
diff --git a/src/components/inner-pages/projects/project-four/ProjectArea.tsx b/src/components/inner-pages/projects/project-four/ProjectArea.tsx
deleted file mode 100644
index 0653e51..0000000
--- a/src/components/inner-pages/projects/project-four/ProjectArea.tsx
+++ /dev/null
@@ -1,104 +0,0 @@
-"use client"
-import { useEffect, useRef, useState } from "react";
-import dynamic from "next/dynamic";
-import project_data from "@/data/inner-data/ProjectData";
-import Image from "next/image";
-import Link from "next/link";
-
-const Fancybox = dynamic(() => import("@/components/common/Fancybox"), { ssr: false });
-
-const ProjectArea = () => {
-  // Init one ref to store the future Isotope object
-  const isotope = useRef<{ arrange?: Function; destroy?: Function } | null>(null);
-  // Store the filter keyword in a state
-  const [filterKey, setFilterKey] = useState("*");
-
-  // Initialize an Isotope object with configs
-  useEffect(() => {
-    if (typeof window !== "undefined") {
-      const Isotope = require("isotope-layout");
-
-      isotope.current = new Isotope(".isotop-gallery-wrapper", {
-        itemSelector: ".isotop-item",
-        layoutMode: "fitRows",
-      });
-
-      // Cleanup
-      return () => {
-        isotope.current?.destroy?.();
-      };
-    }
-  }, []);
-
-  // Handling filter key change
-  useEffect(() => {
-    if (filterKey === "*") isotope.current?.arrange?.({ filter: "*" });
-    else isotope.current?.arrange?.({ filter: `.${filterKey}` });
-  }, [filterKey]);
-
-  const [selectedFilter, setSelectedFilter] = useState("*");
-
-  const handleFilterKeyChange = (key:any) => () => {
-    setFilterKey(key);
-    setSelectedFilter(key);
-  };
-
-   return (
-      <div className="project-section-one mt-150 xl-mt-100 mb-170 xl-mb-100">
-         <div className="container">
-            <div className="filter-nav-one style-two border-line d-lg-flex align-items-center justify-content-between">
-               <ul className="style-none d-flex justify-content-center flex-wrap isotop-menu-wrapper">
-                  <li className={selectedFilter === "*" ? "is-checked" : ""} onClick={handleFilterKeyChange("*")}>All</li>
-                  <li className={selectedFilter === "apartments" ? "is-checked" : ""} onClick={handleFilterKeyChange("apartments")}>Apartments</li>
-                  <li className={selectedFilter === "house" ? "is-checked" : ""} onClick={handleFilterKeyChange("house")}>House</li>
-                  <li className={selectedFilter === "villa" ? "is-checked" : ""} onClick={handleFilterKeyChange("villa")}>VILLA</li>
-                  <li className={selectedFilter === "flat" ? "is-checked" : ""} onClick={handleFilterKeyChange("flat")}>FLAT</li>
-               </ul>
-               <ul className="style-none d-flex align-items-center justify-content-center social-icon md-mt-20">
-                  <li>Share :</li>
-                  <li><Link href="#"><i className="fa-brands fa-facebook-f"></i></Link></li>
-                  <li><Link href="#"><i className="fa-brands fa-twitter"></i></Link></li>
-                  <li><Link href="#"><i className="fa-brands fa-instagram"></i></Link></li>
-               </ul>
-            </div>
-
-            <div id="isotop-gallery-wrapper" className="isotop-gallery-wrapper grid-1column pt-40 lg-pt-30">
-               <div className="grid-sizer"></div>
-               {project_data.filter((items) => items.page === "project_4").map((item) => (
-                  <div key={item.id} className={`isotop-item house flat ${item.category}`}>
-                     <div className="project-block-three mt-80 lg-mt-50">
-                        <div className="row gx-xxl-5 align-items-center">
-                           <div className="col-lg-6">
-                              <figure className="image-wrapper position-relative z-1 overflow-hidden">
-                                 <Fancybox
-                                    options={{
-                                       Carousel: {
-                                          infinite: true,
-                                       },
-                                    }}
-                                 >
-                                    <a className="d-block position-relative" data-fancybox="gallery17" data-caption={item.title} href={`/assets/images/project/img_${item.fancybox}.jpg`}>
-                                       <Image src={item.thumb} alt="" className="w-100 tran5s" />
-                                    </a>
-                                 </Fancybox>
-                              </figure>
-                           </div>
-                           <div className="col-lg-6 ms-auto">
-                              <div className="caption ps-xxl-5">
-                                 <div className="tag fw-500 text-uppercase">{item.category}</div>
-                                 <h3><Link href="/project_details_01">{item.title}</Link></h3>
-                                 <p className="fs-24 pt-45 lg-pt-30 md-pt-10 pb-50 lg-pb-30 md-pb-10">{item.desc}</p>
-                                 <Link href="/project_details_01" className="btn-thirteen rounded-circle"><i className="fa-thin fa-arrow-up-right"></i></Link>
-                              </div>
-                           </div>
-                        </div>
-                     </div>
-                  </div>
-               ))}
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default ProjectArea
diff --git a/src/components/inner-pages/projects/project-four/index.tsx b/src/components/inner-pages/projects/project-four/index.tsx
deleted file mode 100644
index 7894c22..0000000
--- a/src/components/inner-pages/projects/project-four/index.tsx
+++ /dev/null
@@ -1,19 +0,0 @@
-import BreadcrumbThree from "@/components/common/breadcrumb/BreadcrumbThree"
-import HeaderFour from "@/layouts/headers/HeaderFour"
-import FooterFour from "@/layouts/footers/FooterFour"
-import ProjectArea from "./ProjectArea"
-import FancyBanner from "@/components/common/FancyBanner"
-
-const ProjectFour = () => {
-   return (
-      <>
-         <HeaderFour />
-         <BreadcrumbThree title="Our Projects" link="#" link_title="Pages" sub_title="Projects" style={false} />
-         <ProjectArea />
-         <FancyBanner />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ProjectFour;
diff --git a/src/components/inner-pages/projects/project-one/FancyBanner.tsx b/src/components/inner-pages/projects/project-one/FancyBanner.tsx
deleted file mode 100644
index c23445e..0000000
--- a/src/components/inner-pages/projects/project-one/FancyBanner.tsx
+++ /dev/null
@@ -1,23 +0,0 @@
-import Image from "next/image"
-import Link from "next/link"
-
-import titleShape from "@/assets/images/shape/title_shape_08.svg"
-
-const FancyBanner = () => {
-   return (
-      <div className="fancy-banner-three position-relative text-center z-1 pt-140 xl-pt-100 md-pt-80 pb-150 xl-pb-100">
-         <div className="container">
-            <div className="row">
-               <div className="col-xl-7 col-md-8 m-auto">
-                  <div className="title-one mb-45 md-mb-30">
-                     <h2>Any Inquiry? <span>Feel free<Image src={titleShape} alt="" className="lazy-img" /></span> To contact Us.</h2>
-                  </div>
-                  <Link href="/contact" className="btn-five text-uppercase">SEND MESSAGE</Link>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default FancyBanner
diff --git a/src/components/inner-pages/projects/project-one/ProjectArea.tsx b/src/components/inner-pages/projects/project-one/ProjectArea.tsx
deleted file mode 100644
index ef6ebd2..0000000
--- a/src/components/inner-pages/projects/project-one/ProjectArea.tsx
+++ /dev/null
@@ -1,78 +0,0 @@
-"use client"
-import { useEffect, useRef, useState } from "react";
-import dynamic from "next/dynamic";
-import project_data from "@/data/inner-data/ProjectData";
-import Image from "next/image";
-import Link from "next/link";
-
-const Fancybox = dynamic(() => import("@/components/common/Fancybox"), { ssr: false });
-
-const ProjectArea = () => {
-  // Init one ref to store the future Isotope object
-  const isotope = useRef<{ arrange?: Function; destroy?: Function } | null>(null);
-  // Store the filter keyword in a state
-  const [filterKey, setFilterKey] = useState("*");
-
-  // Initialize an Isotope object with configs
-  useEffect(() => {
-    if (typeof window !== "undefined") {
-      const Isotope = require("isotope-layout");
-
-      isotope.current = new Isotope(".isotop-gallery-wrapper", {
-        itemSelector: ".isotop-item",
-        layoutMode: "fitRows",
-      });
-
-      // Cleanup
-      return () => {
-        isotope.current?.destroy?.();
-      };
-    }
-  }, []);
-
-  // Handling filter key change
-  useEffect(() => {
-    if (filterKey === "*") isotope.current?.arrange?.({ filter: "*" });
-    else isotope.current?.arrange?.({ filter: `.${filterKey}` });
-  }, [filterKey]);
-
-  const [selectedFilter, setSelectedFilter] = useState("*");
-
-  const handleFilterKeyChange = (key:any) => () => {
-    setFilterKey(key);
-    setSelectedFilter(key);
-  };
-
-   return (
-      <div className="project-section-one mt-150 xl-mt-100">
-         <div className="container">
-            <div className="filter-nav-one">
-               <ul className="style-none d-flex justify-content-center flex-wrap isotop-menu-wrapper">
-                  <li className={selectedFilter === "*" ? "is-checked" : ""} onClick={handleFilterKeyChange("*")}>All</li>
-                  <li className={selectedFilter === "apartments" ? "is-checked" : ""} onClick={handleFilterKeyChange("apartments")}>Apartments</li>
-                  <li className={selectedFilter === "house" ? "is-checked" : ""} onClick={handleFilterKeyChange("house")}>House</li>
-                  <li className={selectedFilter === "villa" ? "is-checked" : ""} onClick={handleFilterKeyChange("villa")}>VILLA</li>
-                  <li className={selectedFilter === "flat" ? "is-checked" : ""} onClick={handleFilterKeyChange("flat")}>FLAT</li>
-               </ul>
-            </div>
-
-            <div id="isotop-gallery-wrapper" className="isotop-gallery-wrapper grid-3column pt-40 xl-pt-10">
-               <div className="grid-sizer"></div>
-               {project_data.filter((items) => items.page === "project_1").map((item) => (
-                  <div key={item.id} className={`isotop-item ${item.category}`}>
-                     <div className="project-block-one mt-50 lg-mt-30">
-                        <figure className="image-wrapper m0 position-relative z-1 overflow-hidden">
-                           <Image src={item.thumb} alt="" className="w-100" />
-                           <Link href="/project_details_01" className="btn-four inverse rounded-circle"><i
-                              className="bi bi-arrow-up-right"></i></Link>
-                        </figure>
-                     </div>
-                  </div>
-               ))}
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default ProjectArea
diff --git a/src/components/inner-pages/projects/project-one/index.tsx b/src/components/inner-pages/projects/project-one/index.tsx
deleted file mode 100644
index 57e920b..0000000
--- a/src/components/inner-pages/projects/project-one/index.tsx
+++ /dev/null
@@ -1,21 +0,0 @@
-import BreadcrumbOne from "@/components/common/breadcrumb/BreadcrumbOne";
-import HeaderOne from "@/layouts/headers/HeaderOne";
-import ProjectArea from "./ProjectArea";
-import FancyBanner from "./FancyBanner";
-import FooterFour from "@/layouts/footers/FooterFour";
-import FancyBannerCommon from "@/components/common/FancyBanner";
-
-const ProjectOne = () => {
-   return (
-      <>
-         <HeaderOne style={true} />
-         <BreadcrumbOne title="Our Projects" link="#" link_title="Pages" sub_title="Project" style={true} />
-         <ProjectArea />
-         <FancyBanner />
-         <FancyBannerCommon />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ProjectOne;
diff --git a/src/components/inner-pages/projects/project-three/ProjectArea.tsx b/src/components/inner-pages/projects/project-three/ProjectArea.tsx
deleted file mode 100644
index 8730c99..0000000
--- a/src/components/inner-pages/projects/project-three/ProjectArea.tsx
+++ /dev/null
@@ -1,96 +0,0 @@
-"use client"
-import { useEffect, useRef, useState } from "react";
-import dynamic from "next/dynamic";
-import project_data from "@/data/inner-data/ProjectData";
-import Image from "next/image";
-import Link from "next/link";
-
-const Fancybox = dynamic(() => import("@/components/common/Fancybox"), { ssr: false });
-
-const ProjectArea = () => {
-  // Init one ref to store the future Isotope object
-  const isotope = useRef<{ arrange?: Function; destroy?: Function } | null>(null);
-  // Store the filter keyword in a state
-  const [filterKey, setFilterKey] = useState("*");
-
-  // Initialize an Isotope object with configs
-  useEffect(() => {
-    if (typeof window !== "undefined") {
-      const Isotope = require("isotope-layout");
-
-      isotope.current = new Isotope(".isotop-gallery-wrapper", {
-        itemSelector: ".isotop-item",
-        layoutMode: "fitRows",
-      });
-
-      // Cleanup
-      return () => {
-        isotope.current?.destroy?.();
-      };
-    }
-  }, []);
-
-  // Handling filter key change
-  useEffect(() => {
-    if (filterKey === "*") isotope.current?.arrange?.({ filter: "*" });
-    else isotope.current?.arrange?.({ filter: `.${filterKey}` });
-  }, [filterKey]);
-
-  const [selectedFilter, setSelectedFilter] = useState("*");
-
-  const handleFilterKeyChange = (key:any) => () => {
-    setFilterKey(key);
-    setSelectedFilter(key);
-  };
-
-   return (
-      <div className="project-section-one mt-150 xl-mt-100">
-         <div className="container">
-            <div className="filter-nav-one style-two">
-               <ul className="style-none d-flex justify-content-center flex-wrap isotop-menu-wrapper">
-                  <li className={selectedFilter === "*" ? "is-checked" : ""} onClick={handleFilterKeyChange("*")}>All</li>
-                  <li className={selectedFilter === "apartments" ? "is-checked" : ""} onClick={handleFilterKeyChange("apartments")}>Apartments</li>
-                  <li className={selectedFilter === "house" ? "is-checked" : ""} onClick={handleFilterKeyChange("house")}>House</li>
-                  <li className={selectedFilter === "villa" ? "is-checked" : ""} onClick={handleFilterKeyChange("villa")}>VILLA</li>
-                  <li className={selectedFilter === "flat" ? "is-checked" : ""} onClick={handleFilterKeyChange("flat")}>FLAT</li>
-               </ul>
-            </div>
-
-            <div id="isotop-gallery-wrapper" className="isotop-gallery-wrapper grid-2column pt-10">
-               <div className="grid-sizer"></div>
-               {project_data.filter((items) => items.page === "project_3").map((item) => (
-                  <div key={item.id} className={`isotop-item ${item.category}`}>
-                     <div className="project-block-two mt-80 lg-mt-40">
-                        <figure className="image-wrapper m0 position-relative z-1 overflow-hidden">
-                           <div className="tag fw-500 text-uppercase">{item.tag}</div>
-                           <Fancybox
-                              options={{
-                                 Carousel: {
-                                    infinite: true,
-                                 },
-                              }}
-                           >
-                              <a className="d-block position-relative" data-fancybox="gallery17" data-caption={item.title} href={`/assets/images/project/img_${item.fancybox}.jpg`}>
-                                 <Image src={item.thumb} alt="" className="w-100 tran5s" />
-                              </a>
-                           </Fancybox>
-                        </figure>
-                        <div className="caption">
-                           <div className="d-flex justify-content-between align-items-center">
-                              <div className="title">
-                                 <div className="date position-relative">{item.date}</div>
-                                 <Link href="/project_details_01"><h4 className="tran3s">{item.title}</h4></Link>
-                              </div>
-                              <Link href="/project_details_01" className="btn-thirteen rounded-circle"><i className="bi bi-arrow-up-right"></i></Link>
-                           </div>
-                        </div>
-                     </div>
-                  </div>
-               ))}
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default ProjectArea
diff --git a/src/components/inner-pages/projects/project-three/index.tsx b/src/components/inner-pages/projects/project-three/index.tsx
deleted file mode 100644
index 47df367..0000000
--- a/src/components/inner-pages/projects/project-three/index.tsx
+++ /dev/null
@@ -1,21 +0,0 @@
-import BreadcrumbThree from "@/components/common/breadcrumb/BreadcrumbThree"
-import HeaderFour from "@/layouts/headers/HeaderFour"
-import FooterFour from "@/layouts/footers/FooterFour"
-import ProjectArea from "./ProjectArea"
-import FancyBannerCommon from "@/components/common/FancyBanner"
-import FancyBanner from "../project-one/FancyBanner"
-
-const ProjectThree = () => {
-   return (
-      <>
-         <HeaderFour />
-         <BreadcrumbThree title="Our Projects" link="#" link_title="Pages" sub_title="Projects" style={false} />
-         <ProjectArea/>
-         <FancyBanner />
-         <FancyBannerCommon />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ProjectThree;
diff --git a/src/components/inner-pages/projects/project-two/ProjectArea.tsx b/src/components/inner-pages/projects/project-two/ProjectArea.tsx
deleted file mode 100644
index 615d9e7..0000000
--- a/src/components/inner-pages/projects/project-two/ProjectArea.tsx
+++ /dev/null
@@ -1,75 +0,0 @@
-"use client"
-import { useEffect, useRef, useState } from "react";
-import project_data from "@/data/inner-data/ProjectData";
-import Image from "next/image";
-import Link from "next/link";
-
-const ProjectArea = () => {
-  // Init one ref to store the future Isotope object
-  const isotope = useRef<{ arrange?: Function; destroy?: Function } | null>(null);
-  // Store the filter keyword in a state
-  const [filterKey, setFilterKey] = useState("*");
-
-  // Initialize an Isotope object with configs
-  useEffect(() => {
-    if (typeof window !== "undefined") {
-      const Isotope = require("isotope-layout");
-
-      isotope.current = new Isotope(".isotop-gallery-wrapper", {
-        itemSelector: ".isotop-item",
-        layoutMode: "fitRows",
-      });
-
-      // Cleanup
-      return () => {
-        isotope.current?.destroy?.();
-      };
-    }
-  }, []);
-
-  // Handling filter key change
-  useEffect(() => {
-    if (filterKey === "*") isotope.current?.arrange?.({ filter: "*" });
-    else isotope.current?.arrange?.({ filter: `.${filterKey}` });
-  }, [filterKey]);
-
-  const [selectedFilter, setSelectedFilter] = useState("*");
-
-  const handleFilterKeyChange = (key:any) => () => {
-    setFilterKey(key);
-    setSelectedFilter(key);
-  };
-
-   return (
-      <div className="project-section-one mt-150 xl-mt-100">
-         <div className="container">
-            <div className="filter-nav-one">
-               <ul className="style-none d-flex justify-content-center flex-wrap isotop-menu-wrapper">
-                  <li className={selectedFilter === "*" ? "is-checked" : ""} onClick={handleFilterKeyChange("*")}>All</li>
-                  <li className={selectedFilter === "apartments" ? "is-checked" : ""} onClick={handleFilterKeyChange("apartments")}>Apartments</li>
-                  <li className={selectedFilter === "house" ? "is-checked" : ""} onClick={handleFilterKeyChange("house")}>House</li>
-                  <li className={selectedFilter === "villa" ? "is-checked" : ""} onClick={handleFilterKeyChange("villa")}>VILLA</li>
-                  <li className={selectedFilter === "flat" ? "is-checked" : ""} onClick={handleFilterKeyChange("flat")}>FLAT</li>
-               </ul>
-            </div>
-
-            <div id="isotop-gallery-wrapper" className="isotop-gallery-wrapper grid-2column pt-40 xl-pt-10">
-               <div className="grid-sizer"></div>
-               {project_data.filter((items) => items.page === "project_2").map((item) => (
-                  <div key={item.id} className={`isotop-item ${item.category}`}>
-                     <div className="project-block-one mt-50 lg-mt-30">
-                        <figure className="image-wrapper m0 position-relative z-1 overflow-hidden">
-                           <Image src={item.thumb} alt="" className="w-100" />
-                           <Link href="/project_details_01" className="btn-four inverse rounded-circle"><i
-                              className="bi bi-arrow-up-right"></i></Link>
-                        </figure>
-                     </div>
-                  </div>
-               ))}
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default ProjectArea
diff --git a/src/components/inner-pages/projects/project-two/index.tsx b/src/components/inner-pages/projects/project-two/index.tsx
deleted file mode 100644
index ed824f0..0000000
--- a/src/components/inner-pages/projects/project-two/index.tsx
+++ /dev/null
@@ -1,21 +0,0 @@
-import BreadcrumbOne from "@/components/common/breadcrumb/BreadcrumbOne";
-import HeaderOne from "@/layouts/headers/HeaderOne";
-import FooterFour from "@/layouts/footers/FooterFour";
-import FancyBanner from "../project-one/FancyBanner";
-import ProjectArea from "./ProjectArea";
-import FancyBannerCommon from "@/components/common/FancyBanner";
-
-const ProjectOne = () => {
-   return (
-      <>
-         <HeaderOne style={true} />
-         <BreadcrumbOne title="Our Projects" link="#" link_title="Pages" sub_title="Project" style={true} />
-         <ProjectArea />
-         <FancyBanner />
-         <FancyBannerCommon />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ProjectOne;
diff --git a/src/components/inner-pages/services/service-one/BlockFeatureOne.tsx b/src/components/inner-pages/services/service-one/BlockFeatureOne.tsx
deleted file mode 100644
index c9d22b8..0000000
--- a/src/components/inner-pages/services/service-one/BlockFeatureOne.tsx
+++ /dev/null
@@ -1,59 +0,0 @@
-import Image from "next/image"
-import Link from "next/link";
-
-import featureImg_1 from "@/assets/images/assets/screen_11.png";
-import featureImg_2 from "@/assets/images/media/img_54.jpg";
-import featureImg_3 from "@/assets/images/shape/shape_40.svg";
-import featureImg_4 from "@/assets/images/media/img_53.jpg";
-import featureImg_5 from "@/assets/images/shape/title_shape_05.svg";
-
-const BlockFeatureOne = () => {
-   return (
-      <div className="block-feature-eight position-relative z-1 mt-170 xl-mt-120">
-         <div className="container">
-            <div className="position-relative">
-               <div className="row">
-                  <div className="col-lg-6">
-                     <div className="pe-xl-5 wow fadeInLeft">
-                        <div className="row align-items-end">
-                           <div className="col-6">
-                              <div className="media-block position-relative z-1">
-                                 <Image src={featureImg_1} alt="" className="lazy-img screen_03 mb-40" />
-                                 <Image src={featureImg_2} alt="" className="lazy-img main-img w-100" />
-                                 <Image src={featureImg_3} alt="" className="lazy-img shapes shape_01" />
-                              </div>
-                           </div>
-                           <div className="col-6">
-                              <div className="media-block position-relative z-1">
-                                 <Image src={featureImg_4} alt="" className="lazy-img main-img w-100" />
-                              </div>
-                           </div>
-                        </div>
-                     </div>
-                  </div>
-                  <div className="col-lg-6">
-                     <div className="block-two ms-xxl-5 wow fadeInRight">
-                        <div className="bg-wrapper md-mt-60">
-                           <div className="title-one m0">
-                              <h3>120k+  <span>Clients<Image src={featureImg_5} alt="" className="lazy-img" /></span> Rely on our Services.</h3>
-                           </div>
-                           <p className="fs-22 mt-45 mb-60 xl-mb-40 pe-xxl-5">Our founders Dustin Moskovitz and Justin Rosenstein met while leading Engineering .</p>
-                           <ul className="list-style-one fs-22 color-dark style-none">
-                              <li>Loan &amp; low Interest facility</li>
-                              <li>100k+ Property Listing  added & updated</li>
-                              <li>Expert agent consultation</li>
-                           </ul>
-                           <div className="mt-60 lg-mt-40">
-                              <Link href="/contact" className="btn-two">Contact Us</Link>
-                           </div>
-                        </div>
-                     </div>
-                  </div>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default BlockFeatureOne
diff --git a/src/components/inner-pages/services/service-one/BlockFeatureTwo.tsx b/src/components/inner-pages/services/service-one/BlockFeatureTwo.tsx
deleted file mode 100644
index e964469..0000000
--- a/src/components/inner-pages/services/service-one/BlockFeatureTwo.tsx
+++ /dev/null
@@ -1,53 +0,0 @@
-import Image from "next/image"
-import service_data from "@/data/inner-data/ServiceData";
-import Link from "next/link";
-
-import serviceShape_1 from "@/assets/images/shape/title_shape_07.svg";
-import serviceShape_2 from "@/assets/images/shape/shape_73.svg";
-import serviceShape_3 from "@/assets/images/shape/shape_74.svg";
-import serviceShape_4 from "@/assets/images/shape/shape_75.svg";
-import serviceShape_5 from "@/assets/images/shape/shape_76.svg";
-
-const BlockFeatureTwo = ({ style }: any) => {
-   return (
-      <div className={`block-feature-seventeen ${style ? "dark-bg" : "bg-pink-three"} position-relative z-1 pt-120 xl-pt-80 pb-140 xl-pb-80`}>
-         <div className="container">
-            <div className="row">
-               <div className="col-xl-8 m-auto">
-                  <div className="title-one text-center wow fadeInUp mb-40 lg-mb-20">
-                     <h3 className={`${style ? "text-white" : ""}`}>Core <span>Services<Image src={serviceShape_1} alt="" className="lazy-img" /></span></h3>
-                     <p className={`fs-22 color-dark ${style ? "text-white" : ""}`}>Over 745K listings of apartments, lots, plots - available today.</p>
-                  </div>
-               </div>
-            </div>
-            <div className="row gx-xxl-5">
-               {service_data.filter((items) => items.page === "service_1").map((item) => (
-                  <div key={item.id} className="col-lg-4 col-md-6 d-flex mt-40 wow fadeInUp" data-wow-delay="0.1s">
-                     <div className="card-style-ten d-flex align-items-start flex-column w-100 h-100">
-                        <div className="icon d-flex align-items-center justify-content-center rounded-circle tran3s">
-                           <Image src={item.icon} alt="" className="lazy-img" /></div>
-                        <h6>{item.title}</h6>
-                        <p>{item.desc}</p>
-                        <Link href="/service_details" className="btn-twelve sm mt-auto">{item.btn}</Link>
-                     </div>
-                  </div>
-               ))}
-            </div>
-         </div>
-         {
-            style ? (
-               <><Image src={serviceShape_4} alt="" className="lazy-img shapes shape_01" />
-                  <Image src={serviceShape_5} alt="" className="lazy-img shapes shape_02" />
-               </>
-            ) :
-               (
-                  <><Image src={serviceShape_2} alt="" className="lazy-img shapes shape_01" />
-                     <Image src={serviceShape_3} alt="" className="lazy-img shapes shape_02" />
-                  </>
-               )
-         }
-      </div>
-   )
-}
-
-export default BlockFeatureTwo
diff --git a/src/components/inner-pages/services/service-one/FancyBanner.tsx b/src/components/inner-pages/services/service-one/FancyBanner.tsx
deleted file mode 100644
index dc95387..0000000
--- a/src/components/inner-pages/services/service-one/FancyBanner.tsx
+++ /dev/null
@@ -1,21 +0,0 @@
-import Link from "next/link"
-
-const FancyBanner = () => {
-   return (
-      <div className="fancy-banner-two position-relative z-1 pt-90 lg-pt-70 pb-90 lg-pb-70">
-         <div className="container">
-            <div className="row align-items-center">
-               <div className="col-lg-8 m-auto text-center">
-                  <div className="title-one mb-50 lg-mb-40">
-                     <h2 className="text-white fs-lg">Apply for loan with low interest.</h2>
-                     <p className="fs-24 text-white">Apply for loan & get up to $20million with low interest</p>
-                  </div>
-                  <Link href="contact" className="btn-six">Get Started</Link>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default FancyBanner
diff --git a/src/components/inner-pages/services/service-one/index.tsx b/src/components/inner-pages/services/service-one/index.tsx
deleted file mode 100644
index 57acdef..0000000
--- a/src/components/inner-pages/services/service-one/index.tsx
+++ /dev/null
@@ -1,29 +0,0 @@
-import BreadcrumbOne from "@/components/common/breadcrumb/BreadcrumbOne"
-import FooterFour from "@/layouts/footers/FooterFour"
-import HeaderOne from "@/layouts/headers/HeaderOne"
-import BlockFeatureOne from "./BlockFeatureOne"
-import HomeBLockFeatureOne from "@/components/homes/home-one/BLockFeatureOne"
-import BlockFeatureTwo from "./BlockFeatureTwo"
-import Feedback from "@/components/homes/home-one/Feedback"
-import FancyBanner from "./FancyBanner"
-import BLockFeatureTwo from "@/components/homes/home-four/BLockFeatureTwo"
-import FancyBannerCommon from "@/components/common/FancyBanner"
-
-const ServiceOne = () => {
-   return (
-      <>
-         <HeaderOne style={true} />
-         <BreadcrumbOne title="Our Services" link="#" link_title="Pages" sub_title="Services" style={false} />
-         <BlockFeatureOne/>
-         <HomeBLockFeatureOne />
-         <BlockFeatureTwo style={false} />
-         <FancyBanner />
-         <Feedback/>
-         <BLockFeatureTwo style={true} />
-         <FancyBannerCommon />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ServiceOne;
diff --git a/src/components/inner-pages/services/service-two/BLockFeatureOne.tsx b/src/components/inner-pages/services/service-two/BLockFeatureOne.tsx
deleted file mode 100644
index 58d56a0..0000000
--- a/src/components/inner-pages/services/service-two/BLockFeatureOne.tsx
+++ /dev/null
@@ -1,29 +0,0 @@
-import Image from "next/image"
-import Link from "next/link"
-
-import featureImg from "@/assets/images/shape/title_shape_10.svg"
-
-const BLockFeatureOne = () => {
-   return (
-      <div className="block-feature-eleven mt-150 xl-mt-100">
-         <div className="container container-large">
-            <div className="row">
-               <div className="col-lg-5">
-                  <div className="title-one md-mb-40">
-                     <h3>1,230+ <span>Companies <Image src={featureImg} alt="" className="lazy-img" /></span>Trust by us.</h3>
-                  </div>
-               </div>
-               <div className="col-xxl-6 col-lg-7 ms-auto">
-                  <p className="fs-24 lh-lg mb-30 color-dark">Your leading real estate advocate, transforming houses into dreams. Trust us to expertly guide you home.  745,000 apartments & home for sell, rent & mortgage.</p>
-                  <div className="d-inline-flex flex-wrap align-items-center">
-                     <Link href="/about_us_02" className="btn-five md rounded-0 mt-20 me-5"><span>More Details</span></Link>
-                     <Link href="/contact" className="btn-three mt-20"><span>Request a Callback</span> <i className="fa-light fa-arrow-right-long"></i></Link>
-                  </div>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default BLockFeatureOne
diff --git a/src/components/inner-pages/services/service-two/BLockFeatureThree.tsx b/src/components/inner-pages/services/service-two/BLockFeatureThree.tsx
deleted file mode 100644
index 7c3b9ea..0000000
--- a/src/components/inner-pages/services/service-two/BLockFeatureThree.tsx
+++ /dev/null
@@ -1,42 +0,0 @@
-import feature_data from "@/data/home-data/FeatureData";
-import Image from "next/image";
-import Link from "next/link";
-
-import titleShape from "@/assets/images/shape/title_shape_10.svg";
-
-const BLockFeatureThree = () => {
-   return (
-      <div className="block-feature-seven position-relative z-1 mt-150 xl-mt-120 lg-mt-100">
-         <div className="container container-large">
-            <div className="position-relative">
-               <div className="text-center wow fadeInUp">
-                  <div className="title-one">
-                     <h3><span>We’r Here <Image src={titleShape} alt="" className="lazy-img" /></span>to help you</h3>
-                     <p className="fs-22 mt-xs">Explore featured properties for sale.</p>
-                  </div>
-               </div>
-
-               <div className="wrapper position-relative z-1 mt-45 lg-mt-20 mb-100 lg-mb-50">
-                  <div className="row gx-xxl-5">
-                     {feature_data.filter((items) => items.page === "home_two_feature_2").map((item) => (
-                        <div key={item.id} className="col-lg-4 col-md-6 wow fadeInUp" data-wow-delay={item}>
-                           <div className="card-style-two overflow-hidden position-relative z-1 mt-30">
-                              <Image src={item.img ? item.img : ""} alt="" className="lazy-img w-100 tran5s" />
-                              <div className="content text-center">
-                                 <div className="btn-line tran3s fw-500 text-uppercase">{item.tag}</div>
-                                 <h4 className="mt-15 mb-35">{item.title}</h4>
-                                 <Link href="/listing_02" className="btn-four rounded-circle m-auto"><i
-                                    className="bi bi-arrow-up-right"></i></Link>
-                              </div>
-                           </div>
-                        </div>
-                     ))}
-                  </div>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default BLockFeatureThree
diff --git a/src/components/inner-pages/services/service-two/BLockFeatureTwo.tsx b/src/components/inner-pages/services/service-two/BLockFeatureTwo.tsx
deleted file mode 100644
index c196c13..0000000
--- a/src/components/inner-pages/services/service-two/BLockFeatureTwo.tsx
+++ /dev/null
@@ -1,44 +0,0 @@
-import Count from "@/components/common/Count"
-import FaqOne from "@/components/common/faq/FaqOne"
-import Image from "next/image"
-import Link from "next/link"
-
-import featureImg from "@/assets/images/assets/screen_05.jpg"
-
-const BLockFeatureTwo = () => {
-   return (
-      <div className="block-feature-ten mt-170 xl-mt-120 md-mt-100 mb-150 xl-mb-100">
-         <div className="container container-large">
-            <div className="row">
-               <div className="col-xxl-5 col-lg-6 ms-auto order-lg-last wow fadeInRight">
-                  <div className="pb-45">
-                     <div className="title-one mb-70 lg-mb-50">
-                        <div className="upper-title">Why US</div>
-                        <h3>Find you Dream Home Easily.</h3>
-                     </div>
-                     <div className="accordion accordion-style-one top-bottom-line p0 mb-80 lg-mb-50" id="accordionOne">
-                        <FaqOne />
-                     </div>
-                     <Link href="/contact" className="btn-five rounded-0">Contact us</Link>
-                  </div>
-               </div>
-               <div className="col-lg-6">
-                  <div className="media-gallery position-relative z-1 h-100 wow fadeInLeft">
-                     <div className="bg" style={{ backgroundImage: `url(/assets/images/media/img_56.jpg)` }}>
-                        <div className="card-style-three p0">
-                           <div className="bg-wrapper text-center">
-                              <h3>0<Count number={7} />+</h3>
-                              <p>Years Experience <br />with proud.</p>
-                           </div>
-                        </div>
-                        <Image src={featureImg} alt="" className="lazy-img screen_01" />
-                     </div>
-                  </div>
-               </div>
-            </div>
-         </div>
-      </div>
-   )
-}
-
-export default BLockFeatureTwo
diff --git a/src/components/inner-pages/services/service-two/index.tsx b/src/components/inner-pages/services/service-two/index.tsx
deleted file mode 100644
index 68c5bdd..0000000
--- a/src/components/inner-pages/services/service-two/index.tsx
+++ /dev/null
@@ -1,27 +0,0 @@
-import BreadcrumbThree from "@/components/common/breadcrumb/BreadcrumbThree"
-import Brand from "@/components/homes/home-five/Brand"
-import FooterFour from "@/layouts/footers/FooterFour"
-import HeaderFour from "@/layouts/headers/HeaderFour"
-import BLockFeatureOne from "./BLockFeatureOne"
-import BlockFeatureThree from "../service-one/BlockFeatureTwo"
-import BLockFeatureTwo from "./BLockFeatureTwo"
-import FancyBanner from "@/components/common/FancyBanner"
-import BLockFeatureThree from "./BLockFeatureThree"
-
-const ServiceTwo = () => {
-   return (
-      <>
-         <HeaderFour />
-         <BreadcrumbThree title="Our Services" link="#" link_title="Pages" sub_title="Services" style={false} />
-         <BLockFeatureOne />
-         <Brand />
-         <BLockFeatureThree />
-         <BlockFeatureThree style={true} />
-         <BLockFeatureTwo />
-         <FancyBanner />
-         <FooterFour />
-      </>
-   )
-}
-
-export default ServiceTwo
